### PR TITLE
Get most of the tests running on node

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by justinfagnani** [comment]
All tests running and passing on Travis and Sauce now.

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
```
Comment (garlicnation) [inline .travis.yml]: 5?

Comment (justinfagnani) [inline .travis.yml]: I'd like to move our support policy to latest LTS and latest stable. It'll probably still work on 5, but the extra latency in testing isn't worth it, IMO.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+    sources:
+    - google-chrome
+    packages:
+    - google-chrome-stable
+  sauce_connect: true
 script:
-  - npm run testNode
+- mocha
+- xvfb-run wct
+- if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then wct -s 'default'; fi
```
Comment (justinfagnani) [inline .travis.yml]: Anyone know how to only run the Sauce tests for one of the node versions? No need to run them twice.

Comment (garlicnation) [inline .travis.yml]: Something like: `node --version | grep v6`?

Comment (garlicnation) [inline .travis.yml]: Even better: TRAVIS_NODE_VERSION

https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

**Comment by garlicnation** [comment]
LGTM

File: .travis.yml (hunk 2 above)
Comment (justinfagnani) [inline .travis.yml]: docs don't say on what it's set to, probably `node --version` I guess

Comment (justinfagnani) [inline .travis.yml]: This appears to work:

```
- if node --version | grep v6; then xvfb-run wct; fi
- if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && node --version | grep v6; then wct -s 'default'; fi
```

File: .travis.yml (hunk 1 above)
Comment (rictic) [inline .travis.yml]: +1

**Pull request merged**